ast.c

```diff
@@ -870,7 +884,10 @@ static void ast_fill_children_ht(HashTable *ht, zend_ast *ast, ast_state_info_t
 }
 
 static void ast_to_zval(zval *zv, zend_ast *ast, ast_state_info_t *state) {
-	zval tmp_zv, children_zv;
+	zval tmp_zv;
+	zend_array *children;
+	zend_object *obj;
+	zend_bool is_decl;
```

Comment:
Unused variable? Also please combine declaration and initialization where possible.

Reply:
>Unused variable?
Will fix, was testing something out with endLine during a test failure and this is no longer used.

> Also please combine declaration and initialization where possible.

php-ast supports php 7.0+ and some ancient OS versions with php 7.0 may be using an older c compiler. I was unclear about what `-std` should this be expecting as a minimum.

Compilation fails with `-std=c90` but passes with `-std=c99` for unrelated reasons, so it might be safe

Reply:
Using C99 is fine :)